Libraries/libs.mk

```diff
@@ -281,4 +281,12 @@ ifeq ($(LIB_CLI), 1)
 LIB_CLI_DIR ?= $(LIBS_DIR)/CLI
 include $(LIB_CLI_DIR)/CLI.mk
 endif
+
+# Unity (Disabled by default)
+# ************************
+LIB_UNITY ?= 0
+ifeq ($(LIB_UNITY), 1)
+LIB_UNITY_DIR ?= $(LIBS_DIR)/Unity
+include $(LIB_UNITY_DIR)/unity.mk
```

Comment:
Have you implemented this yet?  Would be a good place to manage the central stuff